src/Scene/Quadtree.js

```diff
@@ -24,7 +24,7 @@ function Quadtree(type, schemeTile, link) {
     this.schemeTile = schemeTile;
     this.tileType = type;
     this.minLevel = 2;
-    this.maxLevel = 17;
+    this.maxLevel = 18;
```

Comment:
This change is unrelated to the PR (as is the removal of imagery layers).
You should be extra careful to not mix these kind of changes in a refactoring PR.